client-ui/src/app/generated/services.ts

```diff
@@ -0,0 +1,3 @@
+export { ExecutionEventsService } from './services/execution-events.service';
+export { LineageService } from './services/lineage.service';
+export { OperationsService } from './services/operations.service';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [' should be "](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4584725)